package.json

```diff
@@ -8,7 +8,7 @@
   },
   "repository": {
     "type": "git",
-    "url": "https://github.com/trek10inc/serverless-secrets"
```

Comment:
I recommend reverting this for a cleaner PR going to the upstream repo.

Reply:
Modified source code based on the review.